### ISSUE TITLE
Summarizer argument not working properly

### ISSUE THREAD

**Issue by gamer01** [tensorforce/tensorforce#651]
Setting a *summarizer* and *recorder* in the quickstart

https://github.com/tensorforce/tensorforce/blob/84fdaf92b5b6c68078a73307c02e0db2d3fcd619/examples/quickstart.py#L55-L57

with respect to the [documentation](https://tensorforce.readthedocs.io/en/latest/basics/features.html#tensorboard)

results in an error

```
TypeError: tf_loss_per_instance() missing 1 required positional argument: 'states'
```

gym: 0.16
tf: 2.1.0
tensorforce: git version

**Comment by gamer01** [comment]
it is caused by the labels>"losses" argument in the summarizer

```
Agent.create(...
    summarizer=dict(
        directory='data/summaries',
        # list of labels, or 'all'
        labels=['graph', 'entropy', 'kl-divergence', 'losses', 'rewards'],
        frequency=100  # store values every 100 timesteps
        # (infrequent update summaries every update; other configurations possible)
    ), ...
)
```

**Comment by AlexKuhnle** [comment]
You're right, that's a bug, I just fixed it. Let me know if there are still problems.

**Comment by AlexKuhnle** [comment]
I should mention: TensorFlow 2.1 may not work with Tensorforce, at least I had problems with some TensorFlow-internal hard-to-understand exception messages (which means Tensorforce will stick to 2.0.1 for now).

**Comment by gamer01** [comment]
Can confirm the fix **even with tf 2.1**

**Comment by AlexKuhnle** [comment]
Interesting... maybe I should also give TF 2.1 another go, last time I tried it I ran into quite fundamental problems. Thanks for letting me know. :-)
